Add guidance for optional meta packages (#32942)
dotnet/docs: docs/core/distribution-packaging.md

  - **Version:** \<sdk version>
  - **Contains:** (15)

> The following 2 meta packages are optional. They do bring value for end users, as they abstract the top-level package (dotnet-sdk) which simplifies the installation of the full set of .NET packages. These meta packages would reference a specific .NET SDK version.

Reviewer: ```suggestion
The following two meta packages are optional. They bring value for end users in that they abstract the top-level package (dotnet-sdk), which simplifies the installation of the full set of .NET packages. These meta packages reference a specific .NET SDK version.
```

> - `dotnet[major]` - Installs the specific sdk version

Reviewer: ```suggestion
- `dotnet[major]` - Installs the specified SDK version
```

  - **Version:** \<sdk version>
  - **Example:** dotnet7
  - **Dependencies:** `dotnet-sdk-[major].[minor]`

> - `dotnet` - Installs the specific sdk version determined by distros to be the primary, usually the latest available

Reviewer: ```suggestion
- `dotnet` - Installs a specific SDK version determined by distros to be the primary version&mdash;usually the latest available
```

Reviewer: Yes, they should be updated to SDK - thanks.

Author: Thanks! I'll commit your suggestions and make the other change.
